src/sentry/mail/adapter.py

```diff
@@ -108,10 +123,13 @@ def get_sendable_users(self, project):
 
     @staticmethod
     def notify(notification, target_type, target_identifier=None, **kwargs):
-        if target_type == ActionTargetType.RELEASE_MEMBERS:
-            ActiveReleaseAlertNotification(notification, target_type, target_identifier).send()
-        else:
-            AlertRuleNotification(notification, target_type, target_identifier).send()
+        AlertRuleNotification(notification, target_type, target_identifier).send()
+
+    @staticmethod
+    def notify_active_release(notification):
```

Comment:
Instead, we init the appropriate `ActiveReleaseAlertNotification` here and pin the `target_type` to something we already know.